my-app/src/redux/store.js

```diff
@@ -0,0 +1,14 @@
+import { createStore } from 'redux'
```

Comment:
you don't need a different file for the store, it should be in the file with the reducer